docs/examples/frameworks/tensorflow/tensorflow-dataset.ipynb

```diff
@@ -328,6 +328,7 @@
     "    with tf.device('/gpu:0'):\n",
```

Comment:
<p>Why we need this now?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/2135/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
Without it we would get CPU allocator in the Interator. So the situation here is that we get proper GPU allocator but the check for device compatibility fails saying that the op was placed on the CPU. This is why I decided to add is as an option to ignore this check.